src/crab/ebpf_domain.cpp

```
@@ -19,7 +19,16 @@
 
 namespace prevail {
 
-StringInvariant EbpfDomain::to_set() const { return state.to_set() + stack.to_set(); }
+StringInvariant EbpfDomain::to_set() const {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)